src/List.tsx

```diff
@@ -1,40 +1,51 @@
-import Item from "./Item";
+import { ListItem } from "./App";
+import {makeStyles} from 'tss-react'
+
+interface ListProps {
+  title: string;
+  items: Array<ListItem>;
+  handleCheck: (item: ListItem, checked: boolean) => void;
+}
+
+const useStyles = makeStyles()(() => ({
+  container: {
+    display: "flex",
+    flexDirection: "column",
+  }
+}))
```

Comment:
Please do not use inline styles for html elements. I installed TSS-React library, so you can define classes like this and use them like below. As a side effect, finding suitable class names will boost your creativity! 😄

Reply:
unfortunaelty this does not work
AFAIK tss-react is built for mui thus it requires mui and as a result of mui's emotion dependency emotion as well. So in order to use the makeStyles() function you provided i need to install 3 additional libraries( refer [tss-docs](https://docs.tss-react.dev/) ). Even tss library's standalone use requires emotion as well so for the scope of this project addition 4 additional library a bit too much imo considering no other mui component is being used yet. Let's not use it for now. I will make another project with mui then we can use it. I intend this project to contain no additional dependency apart from react itself. 
```zsh
yarn add @mui/material @emotion/styled @emotion/react #Required for MUI
yarn add tss-react
```

Reply:
I think we can achieve the same functionality with plain css like this, wdyt?
```css
.container {
	display: flex;
	flex-direction: column;
}
```
I use simple css rule with class selector